Assets/MixedRealityToolkit/Utilities/Physics/TwoHandMoveLogic.cs

```diff
@@ -61,11 +73,17 @@ public Vector3 Update(Vector3 centroid, Vector3 manipulationObjectPosition)
             var targetDirection = Vector3.Normalize(gazeAngularOffset * newHandDirection);
             targetDirection = CameraCache.Main.transform.TransformDirection(targetDirection);
 
-            // Compute how far away the object should be based on the ratio of the current to original hand distance
-            var currentHandDistance = Vector3.Magnitude(newHandPosition - pivotPosition);
-            var distanceRatio = currentHandDistance / handRefDistance;
-            var distanceOffset = distanceRatio > 0 ? (distanceRatio - 1f) * DistanceScale : 0;
-            var targetDistance = objRefDistance + distanceOffset;
+            var targetDistance = objRefDistance;
+
+            if(movementConstraint != MovementConstraintType.FixDistanceFromHead)
```

Comment:
we can update the code to a switch when we hit critical mass of constraints (it'll have to change anyhow as we add new ones)

Reply:
Indeed: I already considered a suggestion from Julia to "Constrain to XY plane" and quickly gave up because it would have required more restructuring than I was willing to invest at this moment. I doubt that there are any meaningful constraints that could be efficiently addressed simply by more alternative case handlers.